Docker/README.md

```diff
@@ -0,0 +1,69 @@
+<img src="https://github.com/rawrtc/rawrtc/blob/master/media/rawrtc-icon-256.png" width="48"> + <img src="https://github.com/docker/docker/blob/master/docs/static_files/docker-logo-compressed.png" width="96"> 
+# RAWRTC Build Environment for Docker 
+
+This folder contains a functioning environment to run the RAWRTC build process inside a new Linux based docker instance.
+One of the major advantages of RAWRTC is the removal of non DataChannel functions from the native WebRTC stack, dramatically simplifying the deployment on server based systems. Unlike WebRTC native, RAWRTC does not require media drivers (e.g. Pulse Audio of Fake* Video/Audio drivers) to function. This makes RAWRTC ideal so server side applications which focus on the DataChannel implementation of WebRTC.
```

Comment:
While this is a pretty good sentence to explain the advantages of RAWRTC, I don't think it belongs here. However, I want to have this one in the main `Readme.md`. Could you open a PR to add it there (with *DataChannel* changed to *data channel*) and remove it here?